internal/daemonset/daemonset.go

```diff
@@ -22,8 +22,6 @@ const (
 	kubeletDevicePluginsPath       = "/var/lib/kubelet/device-plugins"
 	nodeLibModulesPath             = "/lib/modules"
 	nodeLibModulesVolumeName       = "node-lib-modules"
```

Comment:
maybe we can restrict it even further to  /lib/modules/<KERNEL_VERSION> ?

Reply:
Done.